pages/common/vault.md

```diff
@@ -0,0 +1,27 @@
+# vault
+
+> HashiCorp Vault, a tool for managing secrets.
+
+- Create a new vault, requiring at least 2 out of 5 keyshares to unseal:
+
+`vault init -key-shares={{5}} -key-threshold={{2}}`
+
+- Unseal a vault, by providing one of the keyshares. Repeat with necessary key-shares until unsealed:
+
+`vault unseal {{key-share-x}}`
+
+- Authenticate client against vault, using an authentication token:
+
+`vault auth {{authentication-token}}`
+
+- Store a new secret in the vault:
+
+`vault write {{secret/hello}} value={{world}}`
```

Comment:
Is the "/" necessary ? Can it be just "hello" ?

Reply:
Vault uses different back-ends (secret, auth, audit, ..), which are modeled as a kind of file system. The 'secret/*' prefix in the example I provided points to their generic back-end to read and write arbitrary secrets.

You can mount other types of back-ends, eg. AWS, PKI, SSH, TOTP, ... but that would be too much to demonstrate in a tldr.

Reply:
I see. So I recommend changing the description to `Store a new secret in the vault using the generic back-end called secret`

And move out the "secret" part out of the tokens- `vault write secret{{/hello}} value={{world}}`

Reply:
Makes sense: Fixed!